requirements.txt

```diff
@@ -48,7 +48,9 @@ pyparsing==2.2.0
 cairocffi
 git+git://github.com/graphite-project/whisper.git#egg=whisper
 git+git://github.com/graphite-project/ceres.git#egg=ceres
-whitenoise
+whitenoise==3.3.1
```

Comment:
why version 3.3.1? what's the error without 3.3.1?

Reply:
`======================================================================                                                                                                  
ERROR: tests.test_user_util (unittest.loader.ModuleImportFailure)                                                                                                       
----------------------------------------------------------------------                                                                                                  
ImportError: Failed to import test module: tests.test_user_util                                                                                                         
Traceback (most recent call last):                                                                                                                                      
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests                                                                                                
    module = self._get_module_from_name(name)                                                                                                                           
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name                                                                                      
    __import__(name)                                                                                                                                                    
  File "/home/vivekpathare/Workspace/zillow/graphite-web/webapp/tests/test_user_util.py", line 8, in <module>                                                           
    from graphite.wsgi import application  # NOQA makes sure we have a working WSGI app                                                                                 
  File "/home/vivekpathare/Workspace/zillow/graphite-web/webapp/graphite/wsgi.py", line 29, in <module>                                                                 
    from whitenoise.django import DjangoWhiteNoise                                                                                                                      
  File "/home/vivekpathare/.local/lib/python2.7/site-packages/whitenoise/django.py", line 2, in <module>                                                                
    '\n\n'                                                                                                                                                              
ImportError:                                                                                                                                                            
                                                                                                                                                                        
Your WhiteNoise configuration is incompatible with WhiteNoise v4.0                                                                                                      
This can be fixed by following the upgrade instructions at:                                                                                                             
http://whitenoise.evans.io/en/stable/changelog.html#v4-0
`
The above error occurs while running the tests.
It seems that the version>=4.0 is having the above error. Pinning 3.3.1 to avoid compatibility error.